CMakeLists.txt

```diff
@@ -268,6 +268,17 @@ else()
   message(STATUS "PIO_REARR_ANY rearranger : Setting the local decomposition map length range for SUBSET rearanger (PIO_REARR_ANY_SUBSET_RANGE) to " ${PIO_REARR_ANY_SUBSET_RANGE} " (default)")
 endif()
 
+if(WITH_ADIOS2)
+  if(DEFINED PIO_OVERRIDE_ADIOS_WITH_PNETCDF_REGEX)
```

Comment:
Can you change the name to SPIO_OVERRIDE_ADIOS_WITH_PNETCDF_FNAME_REGEX or something similar ? And IMO the message should mention that you are overriding the type for files that match the regex ...